### PR TITLE
Move package_manager to add_ons and helm installation ansible

### PR DESCRIPTION
**Pull request by dkoshkin**
Moved `package_manager` from `features` to `add_ons`

A new question are also introduced when running kismatic install plan, 
**An alternative to enabled by default and in the comments specify the options, only "helm" for now**:
```
➜  out git:(monitoring) ✗ ./kismatic install plan
Plan your Kubernetes cluster:
=> Number of etcd nodes [3]:
=> Number of master nodes [2]:
=> Number of worker nodes [3]:
=> Number of ingress nodes (optional, set to 0 if not required) [2]:
=> Number of storage nodes (optional, set to 0 if not required) [0]:
=> Number of existing NFS volumes to be attached [0]:
Cluster Add-Ons:
=> Enable package manager (set to 'none' if not required) [helm none]:

Generating installation plan file template with:
- 3 etcd nodes
- 2 master nodes
- 3 worker nodes
- 2 ingress nodes
- 0 storage nodes
- 0 nfs volumes
- enabled cluster package manager: helm

Wrote plan file template to "kismatic-cluster.yaml"
Edit the plan file to further describe your cluster. Once ready, execute the "install validate" command to proceed.
```
Plan file changes:
```
addons:
  package_manager:
    enabled: true
    provider: helm
```

Helm is now installed in `_kubernetes.yaml` with the other "add-ons" no need for it to be a separate workflow.

### CODE REVIEW

File: pkg/install/plan_types.go [hunk 1]
```diff
@@ -139,12 +148,17 @@ type SSHConnection struct {
 	Node      *Node
 }
 
-type Features struct {
+type AddOns struct {
 	PackageManager PackageManager `yaml:"package_manager"`
 }
 
+type Features struct {
```
Comment (alexbrand) [inline pkg/install/plan_types.go]: Can we have a comment saying that this is deprecated? Took me some time to understanding why we still had features in here :)

File: pkg/install/plan_types.go [hunk 2]
```diff
@@ -8,6 +8,14 @@ import (
 	"github.com/apprenda/kismatic/pkg/ssh"
 )
 
+func PackageManagerProviders() []string {
+	return []string{"helm"}
+}
+
+func DefaultPackageManagerProvider() string {
+	return "helm"
```
Comment (alexbrand) [inline pkg/install/plan_types.go]: Any reason for not using a constant for this?

File: pkg/install/plan_test.go [hunk 3]
```diff
@@ -8,3 +8,18 @@ func TestGenerateAlphaNumericPassword(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestReadWtihDeprecated(t *testing.T) {
```
Comment (alexbrand) [inline pkg/install/plan_test.go]: typo

File: pkg/install/plan.go [hunk 4]
```diff
@@ -76,9 +76,23 @@ func (fp *FilePlanner) Read() (*Plan, error) {
...
 	return p, nil
 }
 
+func readDeprecatedFields(p *Plan) {
+	// only set if not already being set by the user
+	// package_manager moved from features: to add_ons: after KET v1.3.3
+	if !p.AddOns.PackageManager.Enabled && p.Features.PackageManager != nil {
+		p.AddOns.PackageManager.Enabled = p.Features.PackageManager.Enabled
+		// KET v1.3.3 did not have a provider field
+		p.AddOns.PackageManager.Provider = DefaultPackageManagerProvider()
```
Comment (alexbrand) [inline pkg/install/plan.go]: I think we need a `ket133PackageManagerProvider` constant to always be `helm`

File: pkg/cli/plan.go [hunk 5]
```diff
@@ -77,6 +78,12 @@ func doPlan(in io.Reader, out io.Writer, planner install.Planner, planFile strin
 		return fmt.Errorf("The number of nfs volumes must be greater than or equal to zero")
 	}
 
+	fmt.Fprintln(out, "Cluster Add-Ons:")
```
Comment (alexbrand) [inline pkg/cli/plan.go]: Can we add a blank line between the node count prompts and the cluster add-on prompts?

**Comment by dkoshkin** [comment]
CI failure is a flake, see the heapster [PR](https://app.snap-ci.com/apprenda/kismatic/pull/536) that is branched from this code for valid tests.

**Comment by alexbrand** [comment]
LGTM!

**Pull request merged**